user/main.py

```diff
@@ -16,7 +18,53 @@ def get_db():
     finally:
         db.close()
 
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+    
+# To insert new user into the table
+
+@app.post("/create", status_code=status.HTTP_201_CREATED)
+def create_user(request: schemas.User, db: Session=Depends(get_db)):
+    hashed_password = pwd_context.hash(request.password)
+    new_user = models.User(name=request.name, email=request.email, password=hashed_password)
+    if new_user:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST,detail="User already exists")
+    else:
```

Comment:
Shall we skip the else statement here. Which means we can directly mention db.add(new_user). Since on raising an exception will break the flow. So if the code reached db.add(). It must have't passed the if new_user condition.

Also, whenever you are initializing, there will be a User object created. So no need to have the if condition at all.